src/devices/tuya.ts

```diff
@@ -2001,6 +2001,35 @@ const definitions: Definition[] = [
          * Initially wrapped in a try catch: https://github.com/Koenkk/zigbee2mqtt/issues/6313
          */
     },
+    {
+        fingerprint: tuya.fingerprint('TS004F', ['_TZ3000_nuombroo']),
+        model: 'TS004F',
+        vendor: 'TuYa',
+        description: 'Wireless switch with 4 buttons',
+        exposes: [
+            e.battery(), 
+            e.enum('operation_mode', ea.ALL, ['command', 'event']).withDescription(
+                'Operation mode: "command" - for group control, "event" - for clicks'),
+            e.action(['on', 'off', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 
+                'brightness_move_down', '1_single', '1_double', '1_hold', '2_single', '2_double', '2_hold',
+                '3_single', '3_double', '3_hold', '4_single', '4_double', '4_hold'])],
```

Comment:
Do the actions depend on the `operation_mode`? (`command = `on`/`off`, etc, `event` = `1_double`, etc.)?
If this is the case my previous statement was wrong and you can merge these.

Reply:
Yeah, exactly. In `command` mode the device will broadcast the specific action messages to the group (`on`, `off` etc) but in `event` mode it sends special frames handled by `tuya_on_off_action` in fromZigbee.  

Regarding the duplicate model error: should I suffix the new model with a `_1` like the different `TS0505B`'s?